narratives/ncov-19B_resurgence.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Resurgence of SARS-CoV-2 19B lineage
```

Comment:
```suggestion
title: Resurgence of SARS-CoV-2 19B Clade and Possible Convergent Evolution
```
In response to Sidney's comment about more specific title - I think that's a nice idea generally but not particularly fixed on this attempt, if there's a better one!